content/pt-br/event-streaming.md

```diff
@@ -0,0 +1,30 @@
+---
+título: Streaming de Eventos
+estado: Concluído
+categoria: conceito
+---
+
+## O que é isso
+
+O streaming de eventos é uma abordagem em que o software envia dados de eventos de um aplicativo para outro para comunicar continuamente o que eles estão fazendo.
```

Comment:
Sounds better to me "o software envia dados de eventos de uma aplicação para outra" instead of "o software envia dados de eventos de um aplicativo para outro".